scss/detail.scss

```diff
@@ -9,21 +13,25 @@
   font-size: 25px;
   font-weight: bold;
   margin-right: 5px;
+  position: relative;
 
   &::before {
     background: $lemon;
-    content: "";
+    content: '';
```

Comment:
why this change?

Reply:
A tool that I used and modified the quotes.